kubemarine/resources/configurations/globals.yaml

```diff
@@ -102,6 +102,18 @@ workaround:
   retries: 10
   delay_period: 5
 
+thirdparties:
```

Comment:
If new default 3rd-party is added, but this section is forgotten to update, check_paas will consider 3rd-party as not default and won't check recommended versions. It will not even fail in this case, right?
On the other hand, we are starting to move the list of default 3rd-parties from defaults to globals and can complete this rework.
* Move the whole section without `sha` from default to globals.
* Add new enrichment procedure before defaults.compile_inventory which will
    * Copy default 3rd-party from global to inventory, if it is not defined by user.
    * Added recommended sha using thirdparties.get_thirdparty_recommended_sha if source is not defined by user.
* check_paas will use the same thirdparties.get_thirdparty_recommended_sha

Reply:
If we forget to add new 3dparty there, kubemarine can't calculate recommended sha for it and will work with it like with custom 3dparty: sha checks will be only if user specify it.
About moving thirdparties from defaults.yaml to globals.yaml: we can do it, of course, but I think, it breaks the appointment of those files:
* defaults.yaml contains default values in cluster.yaml format and those values can be overridden directly.
* globals.yaml contains values constants that are not available for user customization. 
Thirdparties section is more suitable for defaults.yaml for this appointment. 
I think, this is out of scope for this bug and we need additional discussion for it

Reply:
There are few precedents that globals have some defaults that can be redefined. I found logging and `prepull_group_size`.
Ok, if we want to move the discussion to the separate task and PR, we can skip checking of recommended version for now.